source/agora/registry/Server.d

```diff
@@ -84,19 +83,22 @@ public final class NameRegistry: NameRegistryAPI
 
     public override void putValidator (RegistryPayload registry_payload)
```

Comment:
I was thinking a bit more about this:
let's say an attacker(node A) with a pubkey A signs the payload and the payload contains
1. the attacker pubkey A
2. UTXO X where UTXO X is actually a UTXO for node B

Currently we only check if the payload is properly signed, but we never check if UTXO X belongs to node B or node A

For this reason it might make sense to include the pubkey into that hash(or add the extra check as above)